### PR TITLE
Rb24

### PR DESCRIPTION
**Pull request by carolinedav**
Updated `run_config.py` to stop premature phase advancement.

### CODE REVIEW

File: run_brer/run_config.py [hunk 1]
```diff
@@ -57,8 +57,8 @@ def __init__(self, tpr, ensemble_dir, ensemble_num=1, pairs_json='pair_data.json
         self.run_data = RunData()
         self.run_data.set(ensemble_num=ensemble_num)
 
-        self.state_json = '{}/mem_{}/state.json'.format(ensemble_dir,
```
[the remark this answers is not in the document]
Comment (carolinedav) [reply run_brer/run_config.py:60]: I believe this was an older push that does not relate to #19 but I'm happy to reexamine what we were attempting to complete in #24!

**Comment by carolinedav** [comment]
I do! I cloned and forked the `brer_plugin` repo and added the changes there, pushed it to a new `cnd7cy` branch (let me know if you want me to change the branch name, I was unsure of the github forked repo ~ lingo ~)

File: run_brer/run_config.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (eirrgang) [reply run_brer/run_config.py:60]: I've separated the pr for #24 and #19 now.

**Comment by carolinedav** [comment]
We resolved #19, which I believe this PR is attributed to. Could we remove/close this PR?

**Comment by eirrgang** [comment]
> We resolved #19, which I believe this PR is attributed to. Could we remove/close this PR?

Two fixes got mixed up in the source branch, but I think they've been disentangled. However, there isn't much left of the change targeting #24, and both the conversation and git history are muddled at this point. I'll close the PR and remove the branch, and we can make a fresh start. :-)

**Pull request closed without merging**